### DISCUSSION TITLE
Allow for pattern matching in branch specific environment variables

### DISCUSSION THREAD

**Discussion by SimplyComplexable** [vercel/community#2131] [Idea]
When creating a new environment variable in Vercel for preview environments you can currently specify a git branch, so that the variable will only be set to that value on that specific branch. Building off of this, it would be awesome if you could specify a group of branches using pattern matching. For example `bugfix/**` or `feature/big-release/**`.

**Comment by hmcpadden** [comment]
This is a nice feature request!

**Comment by jrnxf** [comment]
would love this so much!

**Comment by aderopoa** [comment]
Was just about to request for this.

**Comment by felixeichler** [comment]
Yes please! 👍
